src/pages/Chat/Chat.jsx

```diff
@@ -1,3 +1,5 @@
+import { MessageCollection } from '../../components';
+
 export const ChatPage = () => {
-    return <div>This is chat page</div>;
+    return <MessageCollection />;
```

Comment:
actually the data should be passed to MessageCollection, we won't have access to data directly in message collection, we would have access to them in screen

Reply:
Done